### DISCUSSION TITLE
Output number of user that exist in the database

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [pocketbase/pocketbase#2214] [Q&A] [answered]
Hi,

Is there a simple way to output the number of user in the database.

**Comment by Xyphyn** [comment, marked as answer]
You could do

```js
const results = pocketbase.collection('users').getList(1, 1)

results.totalItems // the total amount of users in the db
```

**Reply by MuslemRahimi** [reply]
this gives me the result of 1 even though i have 3 user in my database

**Reply by ganigeorgiev** [reply]
@MuslemRahimi `getList` will return the users and count of only those users you have "List API" rule access. If you want to count all users, no matter of the "List API" rule, then you can create a "View collection" with a custom select query as the other suggested answer below.

**Comment by danibjor** [comment]
you could create a view collection with following:

`SELECT
  (ROW_NUMBER() OVER()) as id,
  count(users.id) as total
FROM users `
